lua/calcium/config.lua

```diff
@@ -5,6 +5,11 @@ M.defaults = {
 	default_mode = "append",
 	scratchpad = {
 		border = "rounded",
+		virtual_text = {
+			format = " = %s",
```

Comment:
Remove the space due to changes done during the syntax highlighting PR, see 284c4ce9f4c9fe9a5a85d11063e67fa590f5e0a2
```suggestion
			format = "= %s",
```